src/test/java/org/mockito/internal/runners/util/RunnerProviderTest.java

```diff
@@ -17,7 +17,7 @@ public void shouldCreateRunnerInstance() throws Throwable {
         //given
         RunnerProvider provider = new RunnerProvider();
         //when
-        RunnerImpl runner = provider.newInstance("org.mockito.internal.runners.SilentJUnitRunner", this.getClass(), null);
+        InternalRunner runner = provider.newInstance("org.mockito.internal.runners.DefaultInternalRunner", this.getClass(), null);
```

Comment:
Since the runner is in the same package as the factory I don't see why it should be a String ?